### ISSUE TITLE
latest libpcap release does still not recognize musl crosscompiler triple

### ISSUE THREAD

**Issue by rofl0r** [the-tcpdump-group/libpcap#373]
this could be fixed by using a recent autoconf version and autoreconf -i, so config.sub gets updated.

currently this workaround is needed

```
 [ -n "$CROSS_COMPILE" ] && \
 xconfflags="--host=$($CC -dumpmachine|sed 's/musl/gnu/') --with-pcap=linux"
./configure $xconfflags ....
```

note the sed that replaces musl with gnu in the toolchain triple to make the hiccup go away.

**Comment by infrastation** [comment]
Could you please provide exact steps to reproduce if the issue still stands for the current master branch of libpcap?

**Comment by rofl0r** [comment]
if `grep musl config.sub` outputs something, we're all set. if not you can get the actual version from http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub
thanks!

**Comment by guyharris** [comment]
Fixed in 5937d86587f1902b65e2f78528491e2d1e348bf0, it appears - it updated config.sub, and:

    $ grep musl config.sub
      linux-musl* | linux-uclibc* | uclinux-uclibc* | uclinux-gnu* | kfreebsd*-gnu* | \
                  | -linux-newlib* | -linux-musl* | -linux-uclibc* \
